spec/models/tabular_import_spec.rb

```diff
@@ -5,24 +5,24 @@
 # Holds specs for code contained in TabularImport (parent class).
 describe TabularImport do
   # Fake child class.
-  class Foo < TabularImport
+  class TestImport < TabularImport
     protected
 
     def process_data
-      sheet.row(1) # Need to actually access some data to trip some errors.
+      sheet.row[0] # Need to actually access some data to trip some errors.
     end
   end
 
   let(:file) { tabular_import_fixture(filename) }
-  let(:import) { Foo.new(file: file).tap(&:run) }
+  let(:import) { TestImport.new(file: file).tap(&:run) }
   let(:run_errors) { import.run_errors }
 
   context "with invalid file type" do
     let(:filename) { "not_tabular.jpg" }
 
     it "should correctly report error" do
       expect(import).not_to be_succeeded
-      expect(run_errors).to eq(["The uploaded file must be a valid XLSX-type Excel file."])
+      expect(run_errors).to eq(["There was an issue with your CSV file: Invalid byte sequence in UTF-8 in line 1."])
```

Comment:
Not a friendly error message, but users should never see this because their upload will be rejected earlier based on content type. They will see `The uploaded file was not an accepted format.`